client/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Suspense } from 'react';
-import { render } from 'react-dom';
-import { hot } from 'react-hot-loader/root';
+// import { render } from 'react-dom';
```

Comment:
Why not just remove these lines instead of commenting out?
(same applies throughout the change)